tests/lib/kubernetes/caasp.py

```diff
@@ -90,3 +93,14 @@ def join(self, nodes: List[NodeBase]):
     def install_kubernetes(self):
         super().install_kubernetes()
         self.join(self.hardware.workers)
+        logger.info("Wait for all nodes to be in 'Ready' state"
+                    "(this may take a while...)")
+        data = {'chdir': self._clusterpath, 'capture': True}
```

Comment:
NIT: why not just adding those arguments directly ? we already have here `attempts` and `interval`. this seems to be unconsistent

Reply:
I think I tried but it was causing a syntax error... Checking again

Reply:
Fixed